OneCommunityGlobal/HighestGoodNetworkApp
# Jay - PERMISSION: Toggle Invisibility Self Others

# Description
This PR enables the user with toggle Invisibility permission to manipulate the visible invisible toggle on the visibility column.

## Related PRS (if any):
This frontend PR is related to the [#735](https://github.com/OneCommunityGlobal/HGNRest/pull/735)  backend PR.

…

## Main changes explained:
- Added conditional logic so that the user with the permission is able to view it on his/her profile page and on other users' profile pages.
…

## How to test (watch video for step by step testing):
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a user without user management permissions such as volunteer or core team -> add Toggle Invisibility Permission Self and Others under User Management -> scroll down then submit
6. log into the account who was just granted the permission and go to its own profile page, click on teams tab, and verify visibility column is visible and toggling the switch works as well as saving (do the same on a different user's profile page)

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/af6b8f52-b066-4901-bf67-eedb58700017



## Note:
**Unmute the video.**
